utils/common/src/test/java/org/apache/brooklyn/util/text/VersionComparatorTest.java

```diff
@@ -56,40 +35,51 @@ public void testSnapshotSuffixComparison() {
 
     @Test
     public void testComparison() {
```

Comment:
BTW the tests would need to be careful to supply actually different String objects, to avoid hitting the `if (Objects.equal(v1, v2)) return 0;` [here](https://github.com/apache/brooklyn-server/pull/743/files#diff-a66acd1b299b8d264a4c2c6f052b27c5R77). We'd want to be sure to exercise the code below that test.